docs/building_the_project.rst

```diff
@@ -379,13 +379,7 @@ With:
 
    -DENABLE_CURAND_BACKEND=True   \
 
-To build with the cuFFT backend instead simply replace:
-
-.. code-block:: bash
-
-   -DENABLE_CUBLAS_BACKEND=True   \
-
-With:
+The CuFFT backend can be enabled in a similar way to the CuBLAS backend, by setting the following CMake variable to `True`:
```

Comment:
May I suggest to merge the note about cuFFT and cuRAND backends into one note to avoid duplication and "misleading" working for the cuRAND backend? I.e. removing the cuRAND part and replacing the cuFFT note with something like "The cuFFT and cuRAND backends can be enabled in a similar way to the cuBLAS backend, by setting the following CMake variable to True" and list both -DENABLE_CUFFT_BACKEND=True and -DENABLE_CURAND_BACKEND=True

Reply:
I've followed your suggestion in [8170ee2](https://github.com/oneapi-src/oneMKL/pull/457/commits/8170ee2eae77968bae6202c3f86af98c6340001b):

> The CuFFT and CuRAND backends can be enabled in a similar way to the CuBLAS backend, by setting the corresponding CMake variable(s) to `True`: